lessons/Data-Science/contracts-tables-visualizations/langs/en-us/pages/functions-for-tables-cont.adoc

```diff
@@ -14,11 +14,11 @@
 
 *Suppose we wanted to know how many animals had 4 legs...*
 
-Type @show{(code '(count animals-table "legs"))} into the Interactions Area and click "Enter"
+Type @show{(code '(count animals-table "legs"))} into the Interactions Area and click "Enter".
 
-@n What did the expression produce? @fitb{}{@ifsoln{a new table showing the counts for the number of animals with 0 legs, 3 legs, 4 legs and 8 legs}}
+@n What did the expression produce? @fitb{}{@ifsoln{a new table showing the frequency of the number of animals with 0 legs, 3 legs, 4 legs and 8 legs}}
 
-@n How many animals had 4 legs? @fitb{}{@ifsoln{29}}
+@n What is the frequency of animals with 4 legs? @fitb{}{@ifsoln{29}}
```

Comment:
I think the old version of this question is better - it forces kids to translate the way we speak to the language of the table, rather than leading them. But I defer to you.

Reply:
How about both? ;-)